css/admin.css

```diff
@@ -341,11 +341,23 @@ div.panel-accordion.show {
     position: absolute; top: 0; bottom: 0;
     right: 117px;
     border: 2px solid #999999; border-radius: 20px;
-    transition: all 0.3s ease-in 0s; 
+    transition: all 0.3s ease-in 0s;
 }
```

Comment:
There should be empty lines between a closing bracket and the next selector. This applies not only to this one line, but to the whole file.